src/pages/dashboardTab/store/dashboardReducer.js

```diff
@@ -584,8 +584,8 @@ export async function fetchAllFileIDsForSelectAll(fileCount = 100000) {
   /* const SELECT_ALL_QUERY = getState().currentActiveTab === tabIndex[1].title
     ? GET_ALL_FILEIDS_FROM_FILESTAB_FOR_ADD_ALL_CART
     : getState().currentActiveTab === tabIndex[1].title
-      ? GET_ALL_FILEIDS_FROM_SAMPLETAB_FOR_ADD_ALL_CART
-      : GET_ALL_FILEIDS_FROM_CASESTAB_FOR_ADD_ALL_CART; */
+      ? GET_ALL_FILEIDS_FROM_SAMPLETAB_FOR_ADD_ALL_CAR0T
```

Comment:
I think there is spelling error in "CAR**0**T". Can you have a look @ogwuazors ?

Reply:
I;ve made the change in a new commit